### DISCUSSION TITLE
Magma 5G Status?

### DISCUSSION THREAD

**Discussion by tayloreisman16** [magma/magma#7803] [General]
Hey all,

I know there is talks to extending Magma to 5G. Do you guys have any good resources on the current status or WIP documentation about Magma 5G?

Thanks,

Taylor

**Comment by karthiksubraveti** [comment]
https://www.magmacore.org/blog/magma-v1.6-release-outlook/

**Reply by dinu-honeys** [reply]
do u mean v1.6 rel is SA supported?

**Comment by alaabouz** [comment]
Hello,
I have similar question regarding the version 1.8 which i have installed recently, any idea if 5G can be enabled as in the nms i can find only eNodeB ?
Thanks!
Alaa.
